k8s/containers/seed-update/changelogs/base/base.yml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
```

Comment:
Why are all Liquibase files copied? They're already in a different folder in Git. Can't they be taken from there?